### PR TITLE
Translations Screen + Some Fixes to Downloader

### PR DESCRIPTION
**Pull request by mohamede1945**
This PR implements the translations screen so that user can download/delete translations. It also fixes some issues with the Downloader.
What still needs to be implemented:
1. Translation Upgrades.
2. Quran Translation Reading View.

### CODE REVIEW

**Comment by mmahalwy** [comment]
wowzah!

File: Quran/SQLiteActiveTranslationsPersistence.swift [hunk 1]
```diff
@@ -13,7 +13,7 @@ struct SQLiteActiveTranslationsPersistence: ActiveTranslationsPersistence, SQLit
 
     let version: UInt = 1
     var filePath: String {
-        return FileManager.default.documentsPath + "/translations.db"
+        return Files.databasesPath + "/translations.db"
```
Comment (ahmedre) [inline Quran/SQLiteActiveTranslationsPersistence.swift]: why not just have a single database with a column for downloaded, may make your life easier?

Comment (ahmedre) [inline Quran/SQLiteActiveTranslationsPersistence.swift]: ah this was old - you already did masha'Allah

Comment (mohamede1945) [inline Quran/SQLiteActiveTranslationsPersistence.swift]: I didn't get it. It's currently a single database.

**Comment by mohamede1945** [comment]
@ahmedre  Thank you so much for your review!
@mmahalwy I can't wait to finish this feature to the end. Actually, most of the app store reviews are around missing translations feature.

**Pull request merged**